Source/Classes/View Controllers/PhotosViewController.swift

```diff
@@ -720,7 +716,7 @@ import MobileCoreServices
             self.currentPhotoIndex = lowIndex
             
             if let photo = self.dataSource.photo(at: lowIndex) {
-                self.delegate?.photosViewController?(self, didNavigateTo: photo, at: lowIndex)
+                self.didNavigateTo(photo: photo, at: lowIndex)
```

Comment:
You missed a line. :)
https://github.com/justindhill/AXPhotoViewer/blob/815582d88adba30851ffd58e6027bb933cd2b45d/Source/Classes/View%20Controllers/PhotosViewController.swift#L725